### PR TITLE
fix:fix proto add underscore problem

### PR DESCRIPTION
**Pull request by haiyux**
#### Other special notes for the reviewers:

fix: #2089 

`kratos proto add hello/hello_world.proto`

```protobuf
syntax = "proto3";

package hello;

option go_package = "/hello;hello";
option java_multiple_files = true;
option java_package = "hello";

service HelloWorld {
	rpc CreateHelloWorld (CreateHelloWorldRequest) returns (CreateHelloWorldReply);
	rpc UpdateHelloWorld (UpdateHelloWorldRequest) returns (UpdateHelloWorldReply);
	rpc DeleteHelloWorld (DeleteHelloWorldRequest) returns (DeleteHelloWorldReply);
	rpc GetHelloWorld (GetHelloWorldRequest) returns (GetHelloWorldReply);
	rpc ListHelloWorld (ListHelloWorldRequest) returns (ListHelloWorldReply);
}

message CreateHelloWorldRequest {}
message CreateHelloWorldReply {}

message UpdateHelloWorldRequest {}
message UpdateHelloWorldReply {}

message DeleteHelloWorldRequest {}
message DeleteHelloWorldReply {}

message GetHelloWorldRequest {}
message GetHelloWorldReply {}

message ListHelloWorldRequest {}
message ListHelloWorldReply {}
```

### CODE REVIEW

File: cmd/kratos/internal/proto/add/add.go [hunk 1]
```diff
@@ -63,7 +65,11 @@ func javaPackage(name string) string {
 }
 
 func serviceName(name string) string {
-	return export(strings.Split(name, ".")[0])
+	return UnderscoreToUpperCamelCase(strings.Split(name, ".")[0])
 }
 
-func export(s string) string { return strings.ToUpper(s[:1]) + s[1:] }
+func UnderscoreToUpperCamelCase(s string) string {
```
Comment (Casper-Mars) [inline cmd/kratos/internal/proto/add/add.go:71]: Add some test?

Comment (Cluas) [inline cmd/kratos/internal/proto/add/add.go:71]: No need to export this.  `UnderscoreToUpperCamelCase` -> `toUpperCamelCase` maybe better.

Comment (haiyux) [reply cmd/kratos/internal/proto/add/add.go:71]: done

Comment (haiyux) [reply cmd/kratos/internal/proto/add/add.go]: done.

File: cmd/kratos/internal/proto/add/add.go [hunk 2]
```diff
@@ -63,7 +65,11 @@ func javaPackage(name string) string {
 }
 
 func serviceName(name string) string {
-	return export(strings.Split(name, ".")[0])
+	return toUpperCamelCase(strings.Split(name, ".")[0])
 }
 
-func export(s string) string { return strings.ToUpper(s[:1]) + s[1:] }
+func toUpperCamelCase(s string) string {
+	s = strings.Replace(s, "_", " ", -1)
```
[the remark this answers is not in the document]
Comment (haiyux) [reply cmd/kratos/internal/proto/add/add.go]: done

**Review by Cluas — APPROVED** [review]
LGTM

**Review by tonybase — APPROVED** [review]

**Pull request merged**